alerts.php

```diff
@@ -449,14 +449,16 @@ function DescribeAlert($alert)
     $device      = dbFetchRow('SELECT hostname, sysName, location, purpose, notes, uptime FROM devices WHERE device_id = ?', array($alert['device_id']));
     $tpl         = dbFetchRow('SELECT `template`,`title`,`title_rec` FROM `alert_templates` JOIN `alert_template_map` ON `alert_template_map`.`alert_templates_id`=`alert_templates`.`id` WHERE `alert_template_map`.`alert_rule_id`=?', array($alert['rule_id']));
     $default_tpl = "%title\r\nSeverity: %severity\r\n{if %state == 0}Time elapsed: %elapsed\r\n{/if}Timestamp: %timestamp\r\nUnique-ID: %uid\r\nRule: {if %name}%name{else}%rule{/if}\r\n{if %faults}Faults:\r\n{foreach %faults}  #%key: %value.string\r\n{/foreach}{/if}Alert sent to: {foreach %contacts}%value <%key> {/foreach}";
-    $obj['hostname']    = $device['hostname'];
-    $obj['sysName']     = $device['sysName'];
-    $obj['location']    = $device['location'];
-    $obj['uptime']      = $device['uptime'];
-    $obj['description'] = $device['purpose'];
-    $obj['notes']       = $device['notes'];
-    $obj['device_id']   = $alert['device_id'];
-    $extra              = $alert['details'];
+    $obj['hostname']     = $device['hostname'];
+    $obj['sysName']      = $device['sysName'];
+    $obj['location']     = $device['location'];
+    $obj['uptime']       = $device['uptime'];
+    $obj['uptime_short'] = formatUptime($device['uptime'], 'short');
+    $obj['uptime_long'] = formatUptime($device['uptime']);
```

Comment:
I'm being very picky, but could you add a single space so it aligns with the other lines? :)